pluma/stream/zeromq.py

```diff
@@ -3,22 +3,22 @@
 
 from pluma.stream import Stream, StreamType
 from pluma.stream.siconversion import SiUnitConversion
-from pluma.io.glia import load_glia
+from pluma.io.zeromq import load_zeromq
 from pluma.sync import ClockRefId
-from pluma.export.streams import export_stream_to_csv
 
 
-class GliaStream(Stream):
-    def __init__(self,
-                 filename: str,
+class ZeromqStream(Stream):
```

Comment:
```suggestion
class ZmqStream(Stream):
```

To improve readability while avoiding issues with capitalization, we could perhaps rename the stream class like this.